src/app/zap-templates/zcl/zcl.json

```diff
@@ -765,8 +758,7 @@
             "LocalVideoRecordingEnabled",
             "LocalSnapshotRecordingEnabled",
             "StatusLightEnabled",
-            "StatusLightBrightness",
-            "FeatureMap"
```

Comment:
Why remove FeatureMap?

Reply:
I reached out via Slack as well,  but I removed FeatureMap from the lists in zcl.json and co. as it's a mandated global element, so it didn't make logical sense (to me at least!) for it to be in here.  Also, if we have FeatureMap, shouldn't we then also add ClusterRevision?
The rules around what does and does not go in here don't seem very clear (or consistently followed!) outside of "structs have to be in here".

Reply:
Re-added FeatureMap and re-ran regen.